packages/eslint-config-godaddy-react/package.json

```diff
@@ -25,26 +26,19 @@
   "bin": {
     "eslint-godaddy-react": "bin/eslint-godaddy-react"
   },
-  "dependencies": {
-    "eslint-config-godaddy": "^4.0.0"
-  },
   "peerDependencies": {
-    "babel-eslint": ">=7.2.1 <11.0.0",
-    "eslint": "^6.1.0",
-    "eslint-plugin-json": "^1.4.0",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-mocha": "^5.3.0",
-    "eslint-plugin-react": "^7.14.2"
+    "eslint": ">=6.0.0"
+  },
+  "dependencies": {
+    "babel-eslint": "^10.0.3",
+    "eslint-config-godaddy": "^4.0.0",
+    "eslint-plugin-flowtype": "^4.5.2",
+    "eslint-plugin-jsx-a11y": "*",
```

Comment:
Could you change this to a `^` version? Don't want a non-backwards-compatible release breaking things for people.